caravel/models.py

```diff
@@ -17,6 +17,7 @@
 import requests
 import sqlalchemy as sqla
 import sqlparse
+import jinja2
```

Comment:
Currently, all packages in `install_requires` in `setup.py` have a specific version. For example, `flask-appbuilder` depends on `werkzeug`, but we specify versions for both packages in `setup.py`. So if we add `Jinja2`, I think we should specify its version for the sake of consistency.